etl/glean_etl.py

```diff
@@ -479,9 +479,15 @@ def write_glean_metadata(output_dir, functions_dir, app_names=None):
 
         # write a search index for the app
         open(os.path.join(functions_dir, f"metrics_search_{app_name}.js"), "w").write(
```

Comment:
nit: Avoid repeting this below by doing something like: 

```python
file_suffix = "fog" if app_name == "firefox_desktop" else app_name
open(os.path.join(functions_dir, f"metrics_search_{file_suffix}.js"), "w").write(
```

Reply:
Yes that's exactly right. Thank you for your patience reading through this work and trying to understand it!